components/ipfs/common/BUILD.gn

```diff
@@ -6,10 +6,13 @@ source_set("common") {
     "ipfs_utils.h",
     "pref_names.cc",
     "pref_names.h",
+    "translate_ipfs_uri.cc",
+    "translate_ipfs_uri.h",
   ]
 
   deps = [
     "//extensions/common",
+    "//net",
```

Comment:
I'm not sure if this dep can cause problems being included in common code.  I guess we aren't actually depending on this common target from common code though so I think it's ok. And will be cleaned up when I get rid of browser and common.